### PR TITLE
Clarify the usage of 'var' keyword in spec guards

### PR DESCRIPTION
**Pull request by fertapric**
Related to https://github.com/elixir-lang/elixir/issues/5800#issuecomment-418693648

### CODE REVIEW

File: lib/elixir/pages/Typespecs.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ If you want to specify more than one variable, you separate them by a comma.
 
     @spec function(arg1, arg2) :: {arg1, arg2} when arg1: atom, arg2: integer
 
-Type variables with no restriction can also be defined.
+Type variables with no restriction can also be defined using the `var` keyword.
```
Comment (josevalim) [inline lib/elixir/pages/Typespecs.md]: I don't like the use of `keyword` here. People may either think it is a special language keyword or the key of a keyword list, which is not the case. Maybe we could say "by giving it the pseudo-type `var`" or similar?

Comment (fertapric) [reply lib/elixir/pages/Typespecs.md]: Yeah. Maybe simplify it to “can also be defined using `var`.”? That way we avoid words like “keyword” and “pseudo-type”?

Comment (josevalim) [reply lib/elixir/pages/Typespecs.md]: Sounds good to me!

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**